avanguard/src/db/models.rs

```diff
@@ -145,6 +147,101 @@ pub fn hash_message<S: AsRef<[u8]>>(message: S) -> [u8; 32] {
     keccak256(&eth_message)
 }
 
+#[derive(Model, Debug)]
+pub struct RefreshToken {
+    pub(crate) id: Option<i64>,
+    pub wallet_id: i64,
+    pub token: String,
+    pub expires_at: NaiveDateTime,
+    pub used_at: Option<NaiveDateTime>,
+    pub blacklisted_at: Option<NaiveDateTime>,
+}
+
+impl RefreshToken {
+    #[must_use]
+    pub fn new(wallet_id: i64, expires_in: u64) -> Self {
+        let expiration = Utc::now() + Duration::seconds(expires_in.try_into().unwrap());
```

Comment:
Nie podobają mi się te unwrapy, generalnie zakładajmy że praktycznie nigdy nie unwrapujemy / expectujemy. Pomyśl czy w tym przypadku nie możemy zamienić Config::refresh_token_timeout (i ten drugi może też?) na u32? Chyba powinno nas to uchronić przed ryzykiem overflowów, ale jeszcze się zastanów bo nie mam czasu myśleć :P.